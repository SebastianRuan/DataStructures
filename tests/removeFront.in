a 5 0
a 12 0
a 333 0
a 99 0
a 111 2
a 35 4
r 0
r 0
a 10 0
r 0
r 0
r 0
r 0
r 0
a 8472 0
a 6 1
