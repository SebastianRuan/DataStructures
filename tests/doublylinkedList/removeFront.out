f d:42 i:-1
f d:3 i:-1
8472
6
